webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
+import InfoCircleIcon from '@patternfly/react-icons/dist/js/icons/info-circle-icon';
```

Comment:
This importing causes an issue during build, please use:
```suggestion
import { InfoCircleIcon } from '@patternfly/react-icons';
```